prow/cluster/jobs/istio/operator/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*  @ayj @costinm @Laci21 @martonsereg @matyix @ostromart @rcernich @sdake @waynz0r
```

Comment:
I don't think this works in subdirectories https://help.github.com/en/articles/about-code-owners#codeowners-file-location

I don't think we should even use this -- if we are going to switch to codeowners then we can do it all in one PR, no need to make it more complicated now